fe/fe-core/src/main/java/org/apache/doris/datasource/maxcompute/MaxComputeExternalCatalog.java

```diff
@@ -67,11 +75,46 @@ public MaxComputeExternalCatalog(long catalogId, String name, String resource, M
         catalogProperty = new CatalogProperty(resource, props);
     }
 
+    //Compatible with existing catalogs in previous versions.
+    protected void generatorEndpoint() {
+        Map<String, String> props = catalogProperty.getProperties();
+
+        if (props.containsKey(MCProperties.ENDPOINT)) {
+            endpoint = props.get(MCProperties.ENDPOINT);
+            return;
+        } else if (props.containsKey(MCProperties.TUNNEL_SDK_ENDPOINT)) {
```

Comment:
Add some example in comment to explain this conversion.

Reply:
ok . i will add it